sport/app/football/model/WallchartJson.scala

```diff
@@ -0,0 +1,45 @@
+package football.model
+
+import model.Competition
+import play.api.libs.json.{Json, Writes}
+import pa.{FootballMatch, LeagueStats, LeagueTableEntry, LeagueTeam, MatchDayTeam, Round, Stage, Venue}
+
+case class WallchartJson(
+    competition: Competition,
+    competitionStages: List[CompetitionStageLike],
+    next: Option[FootballMatch],
+)
+
+object WallchartJson {
+  implicit val roundWrites: Writes[Round] = Json.writes[Round]
+  implicit val stageWrites: Writes[Stage] = Json.writes[Stage]
+  implicit val matchDayTeamWrites: Writes[MatchDayTeam] = Json.writes[MatchDayTeam]
+  implicit val venueWrites: Writes[Venue] = Json.writes[Venue]
+  implicit val footballWrites = new Writes[FootballMatch] {
```

Comment:
Just curious, what prevented `Json.writes` here?

Reply:
They're traits not classes and they're not sealed and they don't have an apply method. If we decide to expose this as an endpoint as is, I'd go into `pa` and set all the write methods at the right level in that package rather than hacking them in here